fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -198,6 +220,10 @@ public PlanFragment visitPhysicalAggregate(
             slotList.addAll(groupSlotList);
             slotList.addAll(aggFunctionOutput);
             outputTupleDesc = generateTupleDesc(slotList, null, context);
+
+            if (needSetSlotToNullable(outputExpressionList)) {
+                setSlotNullable(outputTupleDesc, slotList);
+            }
```

Comment:
if the invoke method is enough simple, please let the code reader understand the core logic without enter the method.
```suggestion
            if (containsGroupingScalarFunctioncOrVirtualSlot(outputExpressionList)) {
                setNullableToNonVirtualSlotReference(outputTupleDesc, slotList);
            }
```